src/libos/src/fs/sefs/sgx_storage.rs

```diff
@@ -10,6 +12,20 @@ use std::sgxfs::{remove, OpenOptions, SgxFile};
 use std::sync::{Arc, SgxMutex as Mutex};
 use std::untrusted::fs;
 
+/// A helper macro to automatically convert a block of code that returns `std::result::Result<T, E1>`
+/// to one that returns `std::result::Result<T, E2>`, where `E1` satisfies `impl From<Error> for E1`
+/// and `E2` satisfies `impl From<Error> for E2`.
```

Comment:
I am sorry that the comments that I have suggested is incorrect. The correct version is
```
/// where `E1` satisfies `impl From<E1> for Error` and `E2` satisfies `impl From<Error> for E2`.
```

Reply:
Done